app/frames/waitlists/[slug]/join/route.tsx

```diff
@@ -394,6 +395,15 @@ const frameHandler = frames(async (ctx) => {
     data: userToAdd,
   });
 
+  // Send the user a direct cast to notify them that they have been waitlisted
+  if (appURL() !== "http://localhost:3000") {
+    const enrichedMessage = `📢🐝\n\nCongratulations!\nYou have succesfully joined ${waitlist.name} (${waitlist.externalUrl}) waitlist.`;
+    const response = await sendDirectCast(fid, enrichedMessage);
```

Comment:
Instead of doing this, I would send the message in our broadcast queue (there should be an endpoint where we send fid and text and it pipes it in a queue)